### PR TITLE
Load journaled cache from Cloud Storage

### PR DESCRIPTION
**Pull request by mdittmer**
`git lfs` quotas on GitHub are creating problems. Serve JDAO caches from
Cloud Storage instead.

### CODE REVIEW

**Comment by mdittmer** [comment]
@arobins PTAL

File: lib/http_jdao.es6.js [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+      documentation: 'Delegate for JDAO.',
+      required: true,
+    },
+    {
+      class: 'Function',
+      name: 'createURL',
+      documentation: 'Platform-independent URL factory function.',
+      factory: function() {
+        const URL = foam.isServer ? require('url').URL : URL;
+        return function(str) { return new URL(str); };
```
Comment (arobins) [inline lib/http_jdao.es6.js]: I think this is causing test failures.

File: lib/http_jdao.es6.js [hunk 2]
```diff
@@ -0,0 +1,123 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+foam.CLASS({
+  package: 'org.chromium.apis.web',
+  name: 'HttpJDAO',
```
[the remark this answers is not in the document]
Comment (mdittmer) [reply lib/http_jdao.es6.js]: Historical accident (should probably be `JDao`, but `DAO` is caps everywhere in FOAM). Also, this is generally how I cope with `TWOACRONYMS` ( => `TwoACRONYMS`).

**Comment by mdittmer** [comment]
@arobins PTAL. Most of this is config changes, but there is a bug fix with overwriting `URL` in the variable scope.

**Pull request merged**